dev/Breadcrumb/Breadcrumb.xaml

```diff
@@ -0,0 +1,66 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.UI.Xaml.Controls">
+
+    <!-- see https://docs.microsoft.com/en-us/windows/uwp/design/style/segoe-ui-symbol-font -->
+    <x:String x:Key="SplitButtonSecondaryButtonContent">&#xE76C;</x:String>
+    <x:String x:Key="SplitButtonSecondaryButtonDefaultContent">&#xE70D;</x:String>
+
+    <Style x:Key="BreadcrumbItemStyle" TargetType="local:BreadcrumbItem">
+
+        <Setter Property="UseSystemFocusVisuals" Value="True" />
+        <Setter Property="FocusVisualMargin" Value="-3" />
+        <Setter Property="IsTabStop" Value="True"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:BreadcrumbItem">
+                    <Border
+                        Background="{TemplateBinding Background}"
+                        BorderBrush="{TemplateBinding BorderBrush}"
+                        BorderThickness="{TemplateBinding BorderThickness}">
+                        <local:SplitButton x:Name="SplitButton" Margin="2">
+                            <ContentPresenter Content="{TemplateBinding Content}" ContentTemplate="{TemplateBinding ContentTemplate}" />
+
+                            <local:SplitButton.Flyout>
+                                <Flyout Placement="Bottom" x:Name="SplitButtonFlyout">
+
+                                    <local:ItemsRepeater x:Name="FlyoutRepeater" HorizontalAlignment="Stretch">
+                                        <local:ItemsRepeater.ItemTemplate>
+                                            <DataTemplate>
+                                                <Button Content="{Binding}" Background="Transparent"/>
+                                            </DataTemplate>
+                                        </local:ItemsRepeater.ItemTemplate>
+                                    </local:ItemsRepeater>
+
+                                </Flyout>
+                            </local:SplitButton.Flyout>
+
+                        </local:SplitButton>
+                    </Border>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style TargetType="local:Breadcrumb">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:Breadcrumb">
+
+                    <local:ItemsRepeater x:Name="BreadcrumbItemRepeater"
```

Comment:
I think all control template parts that are accessed in code-behind should have the "PART_" prefix like in ScrollView.xaml and TwoPaneView.xaml for instance. Old convention.

Reply:
PART_BreadcrumbItem**s**Repeater